modules/jdbc/src/main/java/org/apache/ignite/jdbc/IgniteJdbcDriver.java

```diff
@@ -316,8 +316,9 @@ private static TcpIgniteClient createIgniteClient(
 
                 throw new IgniteClientConnectionException(
                         CONNECTION_ERR,
-                        IgniteStringFormatter.format("Connection to node aborted, "
-                                + "because node doesn't support new JDBC driver [name={}, address={}]", node.name(), node.address()),
+                        IgniteStringFormatter.format("Connection to node aborted, because the node does not support "
+                                + "the feature required by the driver being used. Please use a previous version of the JDBC driver "
```

Comment:
`previous version` sounds to vague. New driver will probably be released in Ignite 3.2. So, if I use driver of version 3.3 and try to connect to server of version 3.1, how could I understand that previous refer to 3.1 and not 3.2?

Reply:
Changed to "compatible" version.